.github/workflows/release.yml

```diff
@@ -0,0 +1,80 @@
+on:
+  push:
+    branches:
+    - main
+    - master
+    tags:
+    - 'v*'
+name: publish Wasm module
+jobs:
+  build:
+    name: Create new release with Wasm artifact
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Prepare Rust environment
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          target: wasm32-unknown-unknown
+      - name: Download wasm-to-oci
+        uses: i3h/download-release-asset@v1
+        with:
+          owner: engineerd
+          repo: wasm-to-oci
+          tag: v0.1.2
+          file: linux-amd64-wasm-to-oci
+      - name: fix wasm-to-oci permissions
+        run: |
+          chmod 755 linux-amd64-wasm-to-oci
+      - name: Build Wasm module
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --target=wasm32-unknown-unknown --release
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: {{ "${{ github.repository_owner }}" }}
+          password: {{ "${{ secrets.GITHUB_TOKEN }}" }}
+      # This step exists because with cargo-generate we cannot make
+      # string substitutions on this template, so we can support
+      # targets with regular dashes that cargo replaces with
+      # underscore.
+      - name: Copy Wasm module to an expected file name
+        run: |
+          cp "target/wasm32-unknown-unknown/release/$(echo "{{project-name}}.wasm" | sed 's/-/_/')" target/wasm32-unknown-unknown/release/{{project-name}}.wasm
+      - name: Publish Wasm policy artifact to OCI registry with the 'latest' tag
+        if: {{ "${{ startsWith(github.ref, 'refs/heads/') }}" }}
+        run: |
+          ./linux-amd64-wasm-to-oci push target/wasm32-unknown-unknown/release/{{project-name}}.wasm ghcr.io/kubewarden/policies/{{project-name}}:latest
```

Comment:
We should probably not leave kubewarden here... wdyt? The only alternative I see is to improve the workflow to use the default organization where the project started out of the template lives currently. That might need some extra steps on the GitHub workflow though. [1]

Reply:
What about making that an GitHub action variable? We could make the default value something like `<TODO replace me with your target repository>`